package.json

```diff
@@ -20,6 +20,7 @@
   "homepage": "https://github.com/ottofeller/hooks#readme",
   "dependencies": {
     "axios": "0.21.0",
+    "intersection-observer": "0.12.0",
```

Comment:
@alexeyjezvaev let's also bump the version of the `@ottofeller/hooks` package from `0.4.6` to `0.5.0`.

Reply:
Minor version updated